### DISCUSSION TITLE
Feature Request: Sound Level

### DISCUSSION THREAD

**Discussion by mustafatoktas** [valinet/ExplorerPatcher#3288] [Ideas]
![Ekran görüntüsü 2024-05-17 224636](https://github.com/valinet/ExplorerPatcher/assets/52534609/df1a3470-39cc-4ec0-b73a-df1d88dbb7d7)

Seeing the volume level on the bar is nice, but could a setting be added to also display the number, like in the second image? So it shows **both** the bar and the number.

![Ekran görüntüsü 2024-05-17 224906](https://github.com/valinet/ExplorerPatcher/assets/52534609/cc402a23-1445-4d8e-8e9b-9e683c4af796)

**Comment by pyrates999** [comment]
This is beyond what EP is for, sorry.
